spec/features/user_views_profile_page_spec.rb

```diff
@@ -3,12 +3,12 @@
 feature "Logged in user views profile page" do
 
   before do
-    user = User.create(first_name: "Donald",
-                            last_name: "Trump",
-                            email: "trump@luxury.com",
-                            username: "therealtrump",
-                            password: "password",
-                            password_confirmation: "password")
+    user = User.create(first_name:           "donald",
+                      last_name:             "trump",
+                      email:                 "trump@luxury.com",
+                      username:              "therealtrump",
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Unnecessary spacing detected.